assets/js/dependencies/async_loader.js

```diff
@@ -1,84 +1,81 @@
 (function ($) {
-
-  var sid = '';
-  var containerType = $('.rstudio-data')[0] ? 'rstudio' : 'web-iframe';
+  var sid = "";
+  var containerType = $(".rstudio-data")[0] ? "rstudio" : "web-iframe";
   var booted = false;
 
   Loader = {
-
     configure: function (containerType) {
       Loader.containerType = containerType;
       /*
       setting up the ajax requests, each request is crossDomain, and it needs to contain the X-Rstudio-session header for cookies and X-Rstudio-Ajax
       to give back the rstudio-layout
       */
       $.ajaxSetup({
-        beforeSend: function (xhr)
-        {
-           xhr.setRequestHeader("X-RStudio-Session", sid);
-           xhr.setRequestHeader("X-RStudio-Ajax",'true');
+        beforeSend: function (xhr) {
+          xhr.setRequestHeader("X-RStudio-Session", sid);
+          xhr.setRequestHeader("X-RStudio-Ajax", "true");
         },
-        cache:false,
-        crossDomain:true
+        cache: false,
+        crossDomain: true,
       });
       /*
       Need to rebind everything when the DOM changes
       */
-      $(document).bind("content-changed",function () {
+      $(document).bind("content-changed", function () {
         Binder.bindGlobalClickHandler();
+        MathJax.typeset();
```

Comment:
Only things that was added in this file?

Reply:
Yes